@xen-orchestra/lite/src/stories/web-core/ui/radio-button/ui-radio-button.story.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      prop('accent')
+        .type('RadioButtonAccent')
+        .required()
+        .enum('info', 'success', 'warning', 'danger')
+        .preset('info')
+        .widget(),
+      prop('disabled').bool().widget(),
+      slot(),
+      slot('info'),
+      setting('defaultSlot').widget(text()).preset('Label'),
+      setting('info').widget(text()).preset('Lorem ipsum dolor sit amet, consectetur adipiscing elit.'),
+    ]"
+  >
+    <div>
+      <UiRadioButton v-model="selectedRadio" :value="1" v-bind="properties">{{ settings.defaultSlot }}</UiRadioButton>
```

Comment:
Since the `value` prop is always of type `string`, it can be written like this:
```suggestion
      <UiRadioButton v-model="selectedRadio" value="1" v-bind="properties">{{ settings.defaultSlot }}</UiRadioButton>
```